server/config/default.yaml

```diff
@@ -171,6 +171,18 @@ postfixBounce:
   # allow connections from localhost only
   host: 127.0.0.1
 
+cas:
+  # enable to use CAS user backend
+  enabled: false
+  url: https://example.cas-server.com
+  # nameTag identifies the attribute to be used for user's full name
+  nameTag: username
+  # mailTag identifies the attribute to be used for user's email address
+  mailTag: mail
+  newUserRole: campaignsAdmin
```

Comment:
Could we have the newUserRole also set to nobody as in the docker entrypoint?

Reply:
Ready https://github.com/Mailtrain-org/mailtrain/pull/960/commits/e980e4d34fe51c864d192599eda5acc5c71831e9